eisbuk-admin/package.json

```diff
@@ -19,6 +19,7 @@
     "formik": "^2.1.5",
     "formik-material-ui": "^3.0.1",
     "i18next-http-backend": "^1.2.6",
+    "jest-environment-jsdom-sixteen": "^2.0.0",
```

Comment:
This is already specified as a dev dependency, and is defnitely not needed at runtime.